keras_cv/src/models/object_detection/faster_rcnn/faster_rcnn.py

```diff
@@ -292,6 +292,7 @@ def __init__(
         rois = _clip_boxes(rois, "yxyx", image_shape)
 
```

Comment:
we could add `faster_rcnn_backbone` in mask_rcnn folder and make these changes there